src/parser/warrior/protection/modules/spells/SpellReflect.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import SCHOOLS from 'game/MAGIC_SCHOOLS';
+
+const debug = true;
```

Comment:
Might want to have this as false in production

Reply:
Oops missed that one